data/token.txt

```diff
@@ -0,0 +1 @@
+REDACTED
```

Comment:
Since `token.txt` does not even hold any token, why is it necessary to have this file? In addition, people might accidentally commit this file too without removing their tokens

Reply:
token.txt _is_ intended to hold a local token only in order to use the testing functions on a local repository. It's only blank here because there isn't any reason (nor is it possible) to push personal tokens to the public repository.

Reply:
It's been a pain constantly commenting and un-commenting the parametrized test cases that switch from the (broken) Travis key to personal keys and then removing said keys to push, so I added a text file to speed up the process.

Reply:
Exactly, for local testing, it would still be better to read in from an environment variable rather than a text file. Because there is no reason to push personal tokens to the public repository, this file shouldn't exist. It gives a chance for people to forget removing the token in the file and commit accidentally to GitHub. But if you think a text file is still the optimal way to do it, you can still do it.

Reply:
Agreed, it is better to read in the environment variable, but the problem is the environment variable doesn't work due to [Github API Rate Limiting](https://developer.github.com/v3/rate_limit/). At the very least, I added this file to the .gitignore, so you can safely push to the repo without accidentally pushing your token.

Reply:
Since one singular token is bound to the environment variable, the 60 requests / hour limit is exceeded almost instantly. Depending on what is decided today, it might be better to remove the token all together since it will almost never have requests remaining for Travis to authenticate with.